src/LeanCode.Contracts/ICommand.cs

```diff
@@ -1,4 +1,4 @@
 namespace LeanCode.Contracts;
 
-[System.Diagnostics.CodeAnalysis.SuppressMessage("?", "CA1040", Justification = "Marker interface.")]
+[System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1040", Justification = "Marker interface.")]
```

Comment:
Please, let's leave `?` everywhere - the `category` introduces unnecessary noise.

Reply:
Counterargument: if `category` is just unnecessary noise, then shouldn't we just switch to `#pragma warning disable` which gives us better control over which lines have warnings suppressed (while also making `Justification` optional)?

Reply:
`pragma warning disable` requires you to have the `restore` in a correct place, which introduces even more noise and it is easy to forget about one.

Also, `Justification` is perfectly fine.

Reply:
Okay. I don't fully agree here but I'll go with `SuppressMessage("?"` because I don't feel like I'll be able to express by reasoning here well enough :upside_down_face: